data/json/snippets/mutant_anatomy.json

```
@@ -268,7 +268,7 @@
       "The chaotic bundle of organs in its torso look much like those of other insects, but as you near the abdomen there's order to the madness: neat rows of venom-filled glands, each feeding into a reservoir at the base of the stinger",
```

Is this indicating that other insects have chaotic bundles of organs?

---

Yes, basically the front looks just as crazypants as the insides of the other weird fauna, but where it really matters form follows function.

---

```suggestion
      "The chaotic bundle of organs in its torso look much like those of other mutated insects, but as you near the abdomen there's order to the madness: comparatively neat rows of venom-filled glands, each feeding into a reservoir at the base of the stinger.",
```
Or "enlarged". "comparatively" could also be rephrased; say, pseudostratified tissue is more organized than a late-stage cancer, but that doesn't mean it's set out as if with a ruler.